beetmoverscript/utils.py

```diff
@@ -309,11 +309,38 @@ def validated_task_id(task_id):
     raise ValueError("No valid taskId found.")
 
 
-def extract_file_config_from_artifact_map(artifact_map, path, task_id, locale):
+def exists_or_endswith(filename, basenames):
+    if isinstance(basenames, tuple):
```

Comment:
I'd be tempted to say:
```
    if isinstance(basenames, str):
        basenames = [basenames]
    for artifact in basenames:
```
as that way we catch all iterables (if someone makes a constant a list later on, for example) and we're not duplicating the check.
That said, it works, so not a blocker.